pyhf/commandline.py

```diff
@@ -0,0 +1,17 @@
+import logging
+logging.basicConfig()
+log = logging.getLogger(__name__)
+
+import click
+import json
+from . import readxml
+
+@click.command()
```

Comment:
what do you think about a toplevel `pyhf` command and `xml2json` could be one of the subcommands

```
> pyhf --help
> pyhf xml2json --help
```


```
@click.group():
@click.option(...) #some global opts
def pyhf():
    pass

@pyhf.command()
@click.option('--entrypoint-xml')
...
def xml2json(...)
```

Reply:
one global option could be the logging verbosity

```
@click.option('verbosity', default='INFO')
def pyhf(verbosity):
    logging.basicConfig(level=getattr(logging, verbosity), format=LOGFORMAT)
 ```

btw I kinda like using log formats with fixed width sections like, thoughts?

```
LOGFORMAT = '%(asctime)s | %(name)20.20s | %(levelname)6s | %(message)s'
```

Reply:
I thought about that too. I'm fine with doing that as well.